integration_tests/testsuite/test_logging_custom.py

```diff
@@ -48,22 +47,28 @@ def runTest(self):
                          'token is {1}, '
                          'level is {2}'.format(log_name, token, level))
 
-            self.assertTrue(self._read_log(client, log_name, token, level),
-                            'Log entry not found for posted token!')
+            project_id = test_util.project_id()
+
+            FILTER = 'resource.type = "global" AND logName = ' \
```

Comment:
ahh interesting. since this is just for optimizing the time it takes to retrieve the log entry from Stackdriver, I'll say let's go ahead and leave it out of this PR for now. I'll open an issue about improving that in the future though, once we start running these tests in different environments (aka GKE) it would be a good thing to look into.